### DISCUSSION TITLE
Tina Cloud Pricing

### DISCUSSION THREAD

**Discussion by scottgallant** [tinacms/tinacms#3372] [General]
We're looking for feedback on the upcoming pricing for Tina Cloud.   We will start charging _new_ teams when Tina Cloud comes out of beta next week. For _existing_ beta users, billing will be enabled on Dec 1.   The pricing is here: 

<img width="779" alt="tina-cloud-pricing" src="https://user-images.githubusercontent.com/776019/199736579-b1cd0916-12bc-40fa-9f75-0d9a7d0ed2de.png">

There was already some discussion about the pricing on [this thread](https://github.com/tinacms/tinacms/discussions/3096#discussioncomment-4001810) , but we'll direct all conversation here going forward.

**Comment by onionhammer** [comment]
I think $599/mo should give you more than 1 site.. Forestry charges $749/mo for unlimited sites, and Forestry includes SSO support. We are a non-profit that is looking to spin out multiple single-purpose sites for multi-year programs (not companies), so this pricing model would be completely untenable.

**Reply by scottgallant** [reply]
Thanks for the feedback.  The $599/month plan does include SSO. 

To give me an idea of what your cost would be, how many sites could you see your non-profit needing in the next 2 or 3 years?   Could you have some of those sites on the Hobby/Team plans, or would they all need to be on the Business Plan (for SSO)?

**Reply by onionhammer** [reply]
We need the content to be manageable by non-technical people, hence the need for a CMS, but the content is rarely updated (maybe once or twice after the site goes live). The content managers are typically the same 2-3 individuals in our organization, but each microsite has its own branding/program terms/domain. We have developed 12 microsites in the past 2 years; mostly around covid-relief programs.

**Comment by niftycodeltd** [comment]
The users, are these client users or do they include devs as well, so for example, if on the Hobby plan doe sthe 2 users relate to a dev and a end user/client or two end users/clients

**Reply by jamespohalloran** [reply]
The user count includes "admins" as well, who are able to both edit through Tina, and manage the project configuration.
(Tina has the advantage of working locally outside of Tina Cloud; so outside of setting up the configuration, a dev on a Tina project won't necessarily _need_ to always be assigned a seat on the project).

**Reply by niftycodeltd** [reply]
Ok, so currently in TinaCloud, I have two admin users but on some sites only one of those users is a collaborator and on others both are a collaborator. If I was to add a 3rd admin but they didn't collaborate on any of the sites, there would be no charge? If that 3rd admin was added to a site that already had to collaborators then there would be a charge? Sorry for the lehmans terms, just need to be sure I have this clear in my mind.

**Reply by jamespohalloran** [reply]
>  I have two admin users but on some sites only one of those users is a collaborator and on others both are a collaborator. If I was to add a 3rd admin but they didn't collaborate on any of the sites, there would be no charge?

That's correct, you're only billed for your project collaborators, not your organization members. So in the following case, you're only billed for one project:

## MyOrg

- Admin1
- Admin2
- Editor1

### Project1 (free)

- Admin1
- Editor1

### Project2 (Team Plan)

- Admin1
- Admin2
- Editor1

We've updated some of our docs here as well:
https://tina.io/docs/tina-cloud/dashboard/users/
https://tina.io/docs/tina-cloud/dashboard/organizations/

Let us know if this is still confusing!

**Comment by suntudo** [comment]
Hey there, just adding my 2c as well. We're a small company and are building out a new monorepo site with landing pages, a blog, and documentation. It's currently split into 3 sites. 

I was really hoping the Tina pricing would be more in line with what it was for Forrestry -- about $30 for a small team that would cover all our sites.  (For those who don't remember. the Starter plan in Forrestry was $29 for 5 teams members + 3 external collaborators for 20 sites + premium support). 

It looks like the free tier will probably cover our immediate use case, but we'll be maxing it out right out of the gate. So if we want to add a 4th collaborator on 3 sites, or add a 4th site, it will go from free to $132. Which is really just way too much for us. At the moment we're just editing everything in Markdown, which is _fine_, but Tina seemed like a great quality of life improvement. But there's no way we can justify that price. Especially considering what we're paying for hosting ($20 on Netlify) and backend (<$5 on Firebase). 

I think the per site fee really stings.

**Comment by Aeolun** [comment]
I already wrote it before, but you either charge by users, or by site. Not both.

**Comment by genox** [comment]
First and foremost, congratulations on going 1.0 to the entire Tina team!

I have voiced my opinion in [the other thread](https://github.com/tinacms/tinacms/discussions/3096) at length already.

Summarised, the current pricing is too aggressive at the low end. Monthly per project _and_ per user fees on small projects that rarely get updated is just not reasonable or defendable in most cases and forcing people to use less accounts leads to security issues with shared passwords and  everything else that follows, too.

Using Tina Cloud from Europe, I also have to deal with massive latency on each request. Up to 1.5s for simple collection retrievals with 10-20 items. This does not make the price look any better, either. Luckily we can use nextjs with stale-while-revalidate and add an Nginx or have Vercel take care of that. I currently proxy Tina API requests locally and cache them, too. Otherwise, the loading times feel extremely long and the experience suffers from it. 

For now, I cannot recommend a paid subscription to anyone who is looking for an economical solution, especially if you are not located in the US or close to one of the DCs.
